src/pybind/mgr/dashboard/frontend/e2e/rgw/buckets.po.ts

```diff
@@ -1,5 +1,183 @@
+import { $, browser, by, element } from 'protractor';
+import { Helper } from '../helper.po';
 import { PageHelper } from '../page-helper.po';
 
-export class BucketsPage extends PageHelper {
-  pages = { index: '/#/rgw/bucket' };
+export class BucketsPageHelper extends PageHelper {
+  static pages = {
+    index: '/#/rgw/bucket',
+    create: '/#/rgw/bucket/create'
+  };
+
+  static navigateTo(page = null) {
```

Comment:
The reason for having a navigateTo in the buckets.po is because the pages are defined in this file and not in the page-helper. We find this easier than using the page-helper navigateTo since you would need to repeatedly pass in the page object as an argument. Would it be okay if we just delete the navigateTo in page-helper and have a navigateTo in each page po file?

Reply:
can you try to do `this.pages =` instead of   `static pages = `;
the helper already has a `pages` variable, so we can just use it.